index.js

```diff
@@ -45,6 +45,30 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
     return arrayOfImportantEvents;
 }
 
+function filterForAuthorAssociation(events = [], association = 'OWNER') {
```

Comment:
I'd like this function name to mention that it's filtering something *out*. So you could call it something like `filterOutByAuthorAssociation` or `rejectByAuthorAssociation`.